tests/antares/services/local_services/test_area.py

```diff
@@ -52,6 +55,18 @@ def test_can_be_created(self, local_study_w_areas):
         created_thermal = local_study_w_areas.get_areas()["fr"].create_thermal_cluster(thermal_name)
         assert isinstance(created_thermal, ThermalCluster)
 
+    def test_duplicate_name_errors(self, local_study_w_thermal):
```

Comment:
Duplicated test inside test_study. This one should be removed i think

Reply:
I'm a bit confused, I don't see a dupe and this seems to be the right place to test this as thermals are part of areas. Could you point me at the duplicated test please?

Reply:
You're right I misread my bad